tracer/src/Datadog.Trace/AppSec/Waf/NativeBindings/WafNative.cs

```diff
@@ -411,19 +411,13 @@ List<string> GetRuntimeSpecificVersions(string path)
                 };
             }
 
-            // treat any path that could contain integrations.json as home folder
-            var integrationsPaths = Environment.GetEnvironmentVariable("DD_INTEGRATIONS")
```

Comment:
@robertpi @anna-git Can you confirm this change in `WafNative.cs`?

Reply:
Yes, the WAF will still work without this.

Reply:
Looks like this was all gone when I rebased, anyway. Thanks!

Reply:
Nope, it just moved to `tracer\src\Datadog.Trace\AppSec\Waf\Initialization\LibraryLoader.cs`, so I'll remove it from there.